tests/utils/exec/idps.go

```diff
@@ -51,7 +52,7 @@ func (idp *IDPService) Init(manifestDirs ...string) error {
 }
 
 func (idp *IDPService) Create(createArgs *IDPArgs, extraArgs ...string) error {
-	createArgs.URL = CON.GateWayURL
+	// createArgs.URL = CON.GateWayURL
```

Comment:
Why commented?

Reply:
`createArgs.URL` is not necessary in order to run the args as Iv'e seen.
but moreover - when we do use it, and there is an idp which require its own URL, the test fail as duplication.

Reply:
Let's delete it is it not necessary and duplication?

Reply:
ack, deleting